config/local/generate_toml.go

```diff
@@ -29,245 +29,473 @@ import (
 	oracletypes "github.com/skip-mev/slinky/x/oracle/types"
 )
 
-var oracleCfgPath = flag.String("oracle-config-path", "oracle.toml", "path to write the oracle config file to")
+var (
+	// oracleCfgPath is the path to write the oracle config file to. By default, this
+	// will write the oracle config file to the local directory.
+	oracleCfgPath = flag.String("oracle-config-path", "oracle.toml", "path to write the oracle config file to")
 
-// LocalConfig defines a readable config for local development. Any changes to this
-// file should be reflected in oracle.toml. To update the oracle.toml file, run
-// `make update-local-config`. This will update any changes to the oracle.toml file
-// as they are made to this file.
-var LocalConfig = config.OracleConfig{
-	// -----------------------------------------------------------	//
-	// -----------------Aggregate Market Config-------------------	//
-	// -----------------------------------------------------------	//
-	Market: config.AggregateMarketConfig{
-		Feeds: map[string]config.FeedConfig{
-			"BITCOIN/USD": {
-				CurrencyPair: oracletypes.NewCurrencyPair("BITCOIN", "USD"),
+	// LocalConfig defines a readable config for local development. Any changes to this
+	// file should be reflected in oracle.toml. To update the oracle.toml file, run
+	// `make update-local-config`. This will update any changes to the oracle.toml file
+	// as they are made to this file.
+	LocalConfig = config.OracleConfig{
+		// -----------------------------------------------------------	//
+		// -----------------Aggregate Market Config-------------------	//
+		// -----------------------------------------------------------	//
+		Market: config.AggregateMarketConfig{
+			Feeds:           Feeds,
+			AggregatedFeeds: AggregatedFeeds,
+		},
+		Production: true,
+		// -----------------------------------------------------------	//
+		// ----------------------Metrics Config-----------------------	//
+		// -----------------------------------------------------------	//
+		Metrics: config.MetricsConfig{
+			Enabled:                 true,
+			PrometheusServerAddress: "0.0.0.0:8002",
+		},
+		UpdateInterval: 1500 * time.Millisecond,
```

Comment:
I increased this from 1s to 1.5s since each price feed (btc/usdt, btc/usdc, etc.) no longer point directly to btc/usd. We should do some more analysis to see how much this new conversion logic interferes with the frequency at which we can expected an aggregated price update for a given price feed.